rviz_visual_testing_framework/include/rviz_visual_testing_framework/visual_test_fixture.hpp

```diff
@@ -41,6 +41,8 @@
 #include "tf2/LinearMath/Quaternion.h"
 #include "tf2_ros/static_transform_broadcaster.h"
 
+#include "ament_index_cpp/get_package_share_directory.hpp"
```

Comment:
Move this include to the .cpp file since the related function isn't used in the header.